### DISCUSSION TITLE
A formatter for HTML files

### DISCUSSION THREAD

**Discussion by vkcku** [biomejs/biome#718] [Ideas]
Are there any plans to provide a formatter for HTML files?

**Comment by unvalley** [comment]
@guacs We are currently working on CSS. Timing and priorities have not yet been determined, but are being considered.

**Comment by vkcku** [comment]
Oh it's great that it's on the roadmap. @unvalley, one more doubt. Do you plan on supporting common HTML template engines like Jinja?

**Reply by unvalley** [reply]
I don't think we should expect anything about specific template engines, etc. at this time. Especially in formatter, we are in need of manpower.

In addition to template engines such as Jinja, modern front-end tools often provide their own syntax, including svelte, vue and astro. I suspect that CSS, HTML, GraphQL, etc. would take precedence over support for these. (I am one of the core-contributors, but I do not represent them)

**Reply by vkcku** [reply]
@unvalley thanks for the response! What you've said makes a lot of sense. Providing support for template engines would be quite a bit of effort and if you have other priorities, that's completely fine. Thanks for the project btw :)

**Reply by unvalley** [reply]
@guacs I forgot to mention [this page](https://biomejs.dev/internals/language-support/#_top) that describes language support. You can track the status. If it looks ok, I will close this discussion🙂

**Reply by vkcku** [reply]
Aah my bad, I really should have searched the documentation in detail :P

I'm totally fine with closing this discussion :)
